### DISCUSSION TITLE
No packages or output file being created for iOS release.

### DISCUSSION THREAD

**Discussion by WenHong-Lau** [unoplatform/uno#17623] [General]
Commands used:
dotnet publish -f net8.0-ios -c Release -o ./publish

it says Created the package: C:\Users\wenhong.lau.DATGEL\source\workspace\UnoApp2\UnoApp2\publish\UnoApp2.ipa but when it is empty folder when I looked at it

**Comment by MartinZikmund** [comment]
Try to add `-p:ArchiveOnBuild=true` to see if the `ipa` file gets created

**Reply by jeromelaban** [reply]
CLI build on Windows is not supported, it's required to do this on a mac.
